### PR TITLE
Release 4.0.1-rc0

### PR DESCRIPTION
**Pull request by JGodin-C2C**
#### Pull Request (PR) description

This is a release pull-request.

### CODE REVIEW

**Comment by JGodin-C2C** [comment]
@bastelfreak  As you are the last person who merged, i guess i should be asking you to review this

**Comment by bastelfreak** [comment]
you need to bump the version in metadata.json to the next major version and then regenerate the changelog. also the tests need to be fixed. I assume they are failing because of a recent breaking change in puppetlabs/firewall.

File: manifests/firewall.pp [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
-      chain  => 'INPUT',
-      proto  => 'tcp',
-      state  => ['NEW'],
-      dport  => $varnish_listen_port,
-      action => 'accept',
+      chain => 'INPUT',
+      proto => 'tcp',
+      state => ['NEW'],
+      dport => $varnish_listen_port,
+      jump  => 'accept',
```
Comment (bastelfreak) [inline manifests/firewall.pp:17]: please do this in it's own PR. In addition we need to update the metadata.json to only allow the latest version where action got replaced with jump.

Comment (JGodin-C2C) [reply manifests/firewall.pp:17]: ok, will do.
